api/v1/errors/SqlError.js

```diff
@@ -0,0 +1,28 @@
+function checkError(err) {
```

Comment:
Here's the alternative I came up with:
We can get rid of the `SQLError` as the error itself (we still need `checkError`), then in `InvalidParameterError` I'll pass in the error SQL generated. I think this is a decent compromise.